### DISCUSSION TITLE
新功能许愿：想要记录穿衣情况

### DISCUSSION THREAD

**Discussion by HaiYian** [WatsonYao/nian_app#103] [Q&A]
想要一个新功能——穿衣记录，可以记录日期（尤其是阴历，因为想参考季节）、温度、穿衣搭配以及感受。支持文字检索和查看往日记录，比如想看看上一次这个温度时穿了什么衣服，可以做个参考，也可以查看哪些衣服穿得多，穿着舒服，为下次购买作参考~
界面可以仿“心情集”吗，写的时候可以选像心情emoji一样的图标，分上几个档，比如寒冷🥶、凉爽🍃、舒适🌿、炎热🥵，这样回顾的时候一目了然哈哈。
不管是否采用，都先谢谢大佬做出这个APP了，太好用了，再次致谢！

**Comment by WatsonYao** [comment]
感谢反馈，这个功能感觉标准记本就可以，标签给到寒冷、舒适等等，进展内容写待搜索文字，还支持图片。
nian 没办法提供阴历，只有公历。

**Reply by HaiYian** [reply]
好的好主意哈哈，标准记本可以写。阴历其实也可以手动输入，谢谢
